File: yurt-system/yurt-system-server/src/main/java/com/github/liulus/yurt/system/repository/UserRoleRepository.java
package com.github.liulus.yurt.system.repository;

import com.github.liulus.yurt.system.model.entity.UserRole;
import com.github.liulus.yurt.jdbc.JdbcRepository;

import java.util.List;

/**
 * @author liulu
 * @version 1.0
 * created_at 2020/5/26
 */
public interface UserRoleRepository extends JdbcRepository<UserRole> {
    List<UserRole> selectByUserId(Long userId);
}

File: yurt-system/yurt-system-server/src/main/kotlin/com/github/liulus/yurt/system/repository/UserRoleRepository.kt
package com.github.liulus.yurt.system.repository

import com.github.liulus.yurt.jdbc.JdbcRepository
import com.github.liulus.yurt.system.model.entity.UserRole

/**
 * @author liulu
 * @version 1.0
 * created_at 2020/5/26
 */
interface UserRoleRepository : JdbcRepository<UserRole?> {
    fun selectByUserId(userId: Long?): List<UserRole?>?
}